code/Controllers/SilverStripeNavigatorItem_Unversioned.php

```diff
@@ -5,11 +5,30 @@
 use SilverStripe\CMS\Controllers\SilverStripeNavigatorItem;
 use SilverStripe\Core\Config\Config;
 use SilverStripe\Core\Convert;
+use SilverStripe\Dev\Deprecation;
+use SilverStripe\ORM\CMSPreviewable;
 use SilverStripe\Security\Member;
 use SilverStripe\Versioned\Versioned;
 
+/**
+ * Class will be moved from `silverstripe/cms` to `silverstripe/admin`.
+ * @deprecated 4.13.0 Will be renamed SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned
+ */
 class SilverStripeNavigatorItem_Unversioned extends SilverStripeNavigatorItem
 {
+    /**
+     * @param DataObject|CMSPreviewable $record
+     */
+    public function __construct(CMSPreviewable $record)
+    {
+        Deprecation::notice(
```

Comment:
Wrap in `Deprecation::withNoReplacement(` - _all_ of these notices need to be wrapped like that, because we don't have a replacement in CMS 4.

Reply:
DONE